assets/sass/main.scss

```diff
@@ -18,6 +18,14 @@ $brand-info: $aplus-info;
 $brand-warning: $aplus-warning;
 $brand-danger: $aplus-danger;
 
+// import new font variables
```

Comment:
The other comments have been started with a capital letter so this one could too.